dex/testing/loadbot/compound.go

```diff
@@ -36,18 +39,26 @@ func runCompound() {
 // runHeavy runs the 'heavy' program, consisting of 2 (12/6) metered
 // sideStackers, 2 (8/4) unmetered sideStackers, and a 5-order sniper.
 func runHeavy() {
-	// Gotta get some major funding to the beta wallets. We'll need something
-	// like 15000 DCR and 150 BTC in the beta nodes.
-	var btcPer uint64 = 150e8 / 10 / 5
-	var dcrPer uint64 = 15000e8 / 10 / 5
+	// Gotta get some major funding to the beta wallets.
+	var base uint64 = 100 * lotSize
+	var quote uint64 = calc.BaseToQuote(uint64(defaultMidGap*1e8), base)
```

Comment:
There are a lot of `1e8` literals I assume are the `calc.RateEncodingFactor`, so should use that if that's the intent.

Reply:
Using a constant now.